src/export_result.py

```diff
@@ -11,15 +11,8 @@ def __init__(self, data):
         self.data = data
 
     def export_to_json(self, file_name):
```

Comment:
Do we need this method at all, it calls another method with the same argument list. Perhaps both  `export_to_json` and `export_to_json_headless` could be the same method accepting `keepArtifact` parameter? Or alternatively `export_to_json_interactive` and `export_to_json_headless` to make the difference more clear